Source/Fuse.Maps/MapTile.uno

```diff
@@ -0,0 +1,298 @@
+using Fuse.Elements;
+using Uno.UX;
+using Uno;
+using Fuse.Maps;
+using Uno.Collections;
+
+namespace Fuse.Controls
+{
+	/** A polyfill for Native @MapView
+
+	This is not meant to be a full implementation, but just something that will help you when developing on local preview.
+
+	This should also work as a standalone component, so it can be used on native.block
+
+	This uses the OSM tile server, and that has both a usage policy https://operations.osmfoundation.org/policies/tiles/ and a license https://www.openstreetmap.org/copyright .
+
+	The mappin is from wikipedia https://commons.wikimedia.org/wiki/File:Map_pin_icon.svg and is Creative Commons Attribution-Share Alike 3.0 Unported. 
+
+	@seealso Fuse.Controls.MapView
+	*/
+
+	public class MapTile : Panel
+	{
+		Fuse.Translation trans;
+		Fuse.Controls.Image[] maps = new Fuse.Controls.Image[9];
+		Fuse.Controls.Grid grid;
+		MarkerIconCache _markerGraphicsCache;
+
+		public MapTile()
+		{
+			grid = new Fuse.Controls.Grid()
+			{
+				ColumnCount = 3
+			};
+			var i = 0;
+			for (var y = 1; y < 4; y++)
+			{
+				for (var x = 1; x < 4; x++)
+				{
+					maps[i] = new Fuse.Controls.Image()
+					{
+						Url = MakeUrl(2, x, y)
+					};
+					grid.Add(maps[i]);
+					i++;
+				}
+			}
+			trans = new Fuse.Translation();
+			var scaling = new Fuse.Scaling()
+			{
+				Factor=1.5f
+			};
+			Children.Add(grid);
+			Children.Add(trans);
+			Children.Add(scaling);
+			Placed += OnPlaced;
+			_markerGraphicsCache = new MarkerIconCache(UpdateMarkers);
+		}
+
+		// http://wiki.openstreetmap.org/wiki/Zoom_levels
+		// http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames
+		String _tileserver = "http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png";
+		public String TileServer
+		{
+			get { return _tileserver; }
+			set { _tileserver = value; }
+		}
+
+		public String MakeUrl (int zoom, int x, int y)
+		{
+			var s = TileServer
+						.Replace("{s}", "{0}")
+						.Replace("{z}", "{1}")
+						.Replace("{x}", "{2}")
+						.Replace("{y}", "{3}");
+			if (x < 0) return null;
+			if (y < 0) return null;
+			if (zoom < 0) return null;
+			return String.Format(s, "a", zoom, x, y);
+		}
+
+		public ObservableList<MapMarker> Markers
+		{
+			get
+			{
+				MapView m = Parent as MapView;
+				if (m != null)
+				{
+					return m.Markers;
+				}
+				return null;
+			}
+		}
+
+		public void ClearMarkers ()
+		{
+			foreach(var i in _markers)
+			{
+				Children.Remove(i);
+			}
+			_markers.Clear();
+		}
+
+		List<Fuse.Controls.Image> _markers = new List<Fuse.Controls.Image>();
+		public void UpdateMarkers()
+		{
+			ClearMarkers();
```

Comment:
Clearing the markers and recreating them on any update is inefficient. Instead of recreating the `Image` obejcts each time could you not reuse the existing ones?